themes/openideal_theme/openideal_theme.theme

```diff
@@ -110,7 +117,9 @@ function openideal_theme_theme_suggestions_comment_alter(array &$suggestions, ar
 function openideal_theme_preprocess_field__created(&$variables) {
```

Comment:
Don't we need to restrict it only to user entity?

Reply:
Yeah, restricted.